src/main/java/frc/robot/supersystems/ArmPose.java

```diff
@@ -0,0 +1,16 @@
+package frc.robot.supersystems;
+
+public class ArmPose {
+    public final double extSetpoint;
+    public final double angleSetpoint;
+    public final double wristSetpoint;
+
+    public ArmPose(double ext, double angle, double wrist) {
+        if (ext < 0.0 || angle < 0.0 || wrist < 0.0) {
+            throw new RuntimeException("Not a valid ArmPose parameter");
```

Comment:
Don't throw. Clamp the value. Don't want to robot to unexpectedly stop while running for a minor issue